docs/schema-migrations.adoc

```diff
@@ -1,9 +1,8 @@
 = How to add a schema migration
 
-1. Copy schema.n.sql to schema.n+1.sql
-2. Modify the schema as you see fit
-3. Write a migrate.n+1.sql that will convert an existing data file in schema.n.sql format to one in schema.n+1.sql format. Note that old migrations must not be modified.
-4. Update kSqlSchemaVersion in sqlstorage.h to n+1
+1. Modify config/sql/schema.sql as you see fit
+2. Write a config/sql/migration/migrate.n+1.sql that will convert an existing data to format in schema.sql. Note that old migrations must not be modified.
+3. Update kSqlSchemaVersion in sqlstorage.h to n+1
```

Comment:
Do we need to update `kSqlSchemaVersion` manually anymore? Couldn't we just use the last version detected by the script?

Reply:
Yes It would be possible if I made small changes.